### ISSUE TITLE
Inproperly aligned labels using text-valign

### ISSUE THREAD

**Issue by faubulous** [cytoscape/cytoscape.js#2755]
We have recently updated to cytoscape.js 3.16.1 and now the vertical align of node labels is broken:

![image](https://user-images.githubusercontent.com/498891/94669287-54627c80-0311-11eb-8da8-99f9db99348f.png)

It seems that there is additional padding inside the label that leads to an of-center alignment of the text. However, I was unable to remove the padding. This is the stylesheet we are using:

```
node {
  shape: round-rectangle;
  text-halign: center;
  text-valign: center;
  text-justification: center;
  text-margin-x: 0;
  text-margin-y: 0;
  text-background-color: blue;
  text-background-opacity: 1;
  text-background-padding: 0;
  font-size: 9px;
  color: black;
  background-color: white;
  border-style: solid;
  border-color: #999;
  border-width: 1px;
  height: 20px;
}
```
Are there any recent changes that might explain this behavior?

**Comment by hasanbalci** [comment]
@faubulous Same with #2749. It should be solved in the master branch and will be available in v.3.16.2.

**Comment by faubulous** [comment]
Alright. I'll give it a try! Thank you. :)

**Comment by maxkfranz** [comment]
Resolved by #2749.  Closing
